README.md

```diff
@@ -174,7 +174,7 @@ Otherwise the client will raise a `Notify.Exceptions.NotifyClientException`:
 ### Email
 
 ```csharp
-EmailNotificationResponse response = client.SendEmail(emailAddress, templateId, personalisation, reference);
+EmailNotificationResponse response = client.SendEmail(emailAddress, templateId, personalisation, reference, replyToId);
```

Comment:
`replyToId` should probably be `emailReplyToId` for consistency

Reply:
Done